verifyFiles.py

```diff
@@ -207,7 +209,7 @@ def read_checksum_from_file(checksums_after_file_path):
          for line in file:
 	     if (len(line.split(' ', 1)) == 2) and '/' in line:
 		 filename=((line.split(' ', 1)[1]).strip())
-		 filename=filename.split('/', 1)[1]
+		 filename=filename.split('/', 2)[2]
```

Comment:
exec/<subject_name>/ ... we are splitting it w.r.t to the occurrence of '/' 2nd time